extensions/interactions/Continue/directives/continue-interaction.directive.html

```diff
@@ -1,4 +1,13 @@
-<md-button ng-if="$ctrl.isInEditorMode" class="oppia-learner-confirm-button">
-  <[$ctrl.buttonText]>
-  <i class="fas fa-arrow-right" style="font-size: 19px; padding-top: 1.5px;"></i>
-</md-button>
+<div class="oppia-interactive-continue">
```

Comment:
This is the continue interaction, right? Rename to 'oppia-interactive-continue-interaction'?

Reply:
Done.